src/main/kotlin/com/jobtracker/api/controllers/models/ContactModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.jobtracker.api.controllers.models
+
+import com.jobtracker.api.repository.entities.ApplicationEntity
+import com.jobtracker.api.repository.entities.ContactEntity
+import com.jobtracker.api.repository.entities.CompanyEntity
+import org.apache.commons.lang3.mutable.Mutable
+import java.util.UUID
+
+data class ContactModel(
+  var contactId: UUID = UUID.randomUUID(),
+  var companyObject: CompanyEntity,
```

Comment:
I don't think that works - I had that originally, but it's of the wrong type. Something needs to be converting somewhere. I could take the ID here, and then use the Repo method to get the object, because it's what the M:1 relationship is expecting...